main.py

```diff
@@ -8,12 +9,13 @@ def main():
     """
     year = "2021"
     cve_year_collection = CVEsCollection(year)
-    cve_dict = cve_year_collection.getCVE_Items()
-    cve_year_collection.getCveByIndex(0)
+    cve = cve_year_collection.getCveByIndex(51)
     print("Hello")
 
```

Comment:
CVEsCollection: Instead of printing the json, should create a function that returns the string, and then the caller will print it. (what if instead of printing it, we'll want to write to a file)